src/pudl/extract/eia861.py

```diff
@@ -41,6 +41,7 @@ def process_raw(self, df, yr, page):
             "The data has not yet been validated, and the structure may change."
         )
         column_map_numeric = self._metadata.get_column_map(yr, page)
+        # print(column_map_numeric)
```

Comment:
can you get rid of this little guy?... or change it to a `logger.debug` with a more informative message if you think it might be helpful in the future.